### PR TITLE
#15 implement database structure

### PR DESCRIPTION
**Pull request by AVM82**
connect to #15

### CODE REVIEW

**Review by kortov — COMMENTED** [review]
Sorry, what means “plu”? I didn’t get it

**Review by ivshapovalov — APPROVED** [review]

**Review by kortov — CHANGES_REQUESTED** [review]
Please rename table fields, there shouldn't be tablenames in them. And may be rename plu table, it's not obvious what it means

**Comment by adaleorion** [comment]
@AVM82  please give some answers for @kortov

File: src/main/resources/sql/create.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "age" SMALLINT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "shop" (
+  "shopId" SERIAL PRIMARY KEY,
+  "shopName" TEXT NOT NULL UNIQUE,
+  "address" TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "plu" (
```
Comment (adaleorion) [inline src/main/resources/sql/create.sql]: I agree with @kortov. This name is not clear . What is plu ?

Comment (AVM82) [reply src/main/resources/sql/create.sql]: Ok, i will do it.

**Review by kortov — APPROVED** [review]

File: src/main/resources/sql/create.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "shop_id" INTEGER NOT NULL,
+  FOREIGN KEY ("shop_id") REFERENCES "shop"("id"),
+  FOREIGN KEY ("buyer_id") REFERENCES "buyer"("id")
+  ON DELETE  CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS "line_order" (
+  "id" SERIAL PRIMARY KEY,
+  "order_id" INTEGER NOT NULL,
+  "goods_id" INTEGER NOT NULL,
```
Comment (kortov) [inline src/main/resources/sql/create.sql]: May be products instead of goods?

**Review by kortov — APPROVED** [review]

**Pull request merged**